noaastn/noaastn.py

```diff
@@ -66,10 +75,10 @@ def get_stations_info(country="all"):
             if i <= skip_lines:
                 continue
             for i in range(len(col_index) - 1):
-                val = line[col_index[i]: col_index[i + 1]].strip()
+                val = line[col_index[i] : col_index[i + 1]].strip()
```

Comment:
Looks like this line is throwing an error on the build action.
```suggestion
                val = line[col_index[i]: col_index[i + 1]].strip()
```